pkg/build/builder/common.go

```diff
@@ -130,6 +130,10 @@ func execPostCommitHook(client DockerClient, postCommitSpec api.BuildPostCommitS
 	if err != nil {
 		return fmt.Errorf("read cgroup limits: %v", err)
 	}
+	parent, err := GetCgroupParent()
```

Comment:
Starting in 3.6, there's no case where this could fail?  I.e. all systems with cgroups we'd run origin on will work?